src/pages/workspace/accounting/qbo/advanced/QuickbooksAccountSelectPage.tsx

```diff
@@ -0,0 +1,90 @@
+import React, {useCallback, useMemo, useState} from 'react';
+import {View} from 'react-native';
+import type {ValueOf} from 'type-fest';
+import HeaderWithBackButton from '@components/HeaderWithBackButton';
+import ScreenWrapper from '@components/ScreenWrapper';
+import SelectionList from '@components/SelectionList';
+import RadioListItem from '@components/SelectionList/RadioListItem';
+import type {ListItem} from '@components/SelectionList/types';
+import Text from '@components/Text';
+import useLocalize from '@hooks/useLocalize';
+import useThemeStyles from '@hooks/useThemeStyles';
+import Navigation from '@libs/Navigation/Navigation';
+import AdminPolicyAccessOrNotFoundWrapper from '@pages/workspace/AdminPolicyAccessOrNotFoundWrapper';
+import FeatureEnabledAccessOrNotFoundWrapper from '@pages/workspace/FeatureEnabledAccessOrNotFoundWrapper';
+import PaidPolicyAccessOrNotFoundWrapper from '@pages/workspace/PaidPolicyAccessOrNotFoundWrapper';
+import withPolicy from '@pages/workspace/withPolicy';
+import type {WithPolicyProps} from '@pages/workspace/withPolicy';
+import CONST from '@src/CONST';
+
+type CustomSelectorTypes = ValueOf<typeof CONST.QBO_SELECTOR_OPTIONS>;
+
+type SelectorType = ListItem & {
+    value: CustomSelectorTypes;
+};
+
+function QuickbooksAccountSelectPage({policy}: WithPolicyProps) {
+    const styles = useThemeStyles();
+    const {translate} = useLocalize();
+
+    const [selectedAccount, setSelectedAccount] = useState<CustomSelectorTypes>(CONST.QBO_SELECTOR_OPTIONS.CROISSANT_CO_PAYROLL_ACCOUNT);
+
+    const policyID = policy?.id ?? '';
+
+    const qboOnlineSelectorOptions = useMemo<SelectorType[]>(
+        () =>
+            Object.entries(CONST.QBO_SELECTOR_OPTIONS).map(([key, value]) => ({
```

Comment:
The list of options are dynamic, and they come from the `policy` object.

Specifically, the list is constructed from the following two values:

`policy.connections.quickbooksOnline.data.bankAccounts` and `policy.connections.quickbooksOnline.data.creditCards`.

You can see the type definitions for these fields in the `onyx/Policy.ts` file.
 
 You can use the `name` field of each `Account` object (you can see the `Policy.ts` file and you know what I'm taking about)